app/views/layouts/application.html.erb

```diff
@@ -24,7 +24,10 @@
       <p><%= t('.language_options') %></p>
       <ul>
         <li><%= link_to 'English', :locale => :en %></li>
-        <li><%= link_to 'Bro', :locale => :bro %></li>
+        <li><%= link_to 'Pseudo', :locale => :pseudo %></li>
+        <li><%= link_to 'Portuguese', :locale => :pt %></li>
+        <li><%= link_to 'Chinese(Traditional)', :locale => :zh_tw %></li>
+        <li><%= link_to 'Chinese(Simplified)', :locale => :zh_ch %></li>   
         <!-- You add links to veiw the page in new locales here-->
```

Comment:
Hi Nathan, 

Thanks for the feedback! I had pushed .yml files Traditional and Simplified Chinese to my Github repo, it should work. 

Thanks, 
Daphne


> On 2015年4月21日, at 下午8:46, nsuitter <notifications@github.com> wrote:
> 
> In app/views/layouts/application.html.erb <https://github.com/miis-l10n/health_blog/pull/8#discussion_r28842600>:
> 
> > @@ -24,7 +24,10 @@
> >        <p><%= t('.language_options') %></p>
> >        <ul>
> >          <li><%= link_to 'English', :locale => :en %></li>
> > -        <li><%= link_to 'Bro', :locale => :bro %></li>
> > +        <li><%= link_to 'Pseudo', :locale => :pseudo %></li>
> > +        <li><%= link_to 'Portuguese', :locale => :pt %></li>
> > +        <li><%= link_to 'Chinese(Traditional)', :locale => :zh_tw %></li>
> > +        <li><%= link_to 'Chinese(Simplified)', :locale => :zh_ch %></li>   
> >          <!-- You add links to veiw the page in new locales here-->
> The links to Chinese(Traditional) and Chinese(Simplified) are broken since there aren't any .yml files that match those locales. I would suggest either removing those two lines or supplying zh_ch and zh_tw resource files.
> 
> —
> Reply to this email directly or view it on GitHub <https://github.com/miis-l10n/health_blog/pull/8/files#r28842600>.
>